kolibri/plugins/learn/assets/src/views/EmbeddedSidePanel/index.vue

```diff
@@ -55,7 +57,10 @@
             :text="coreString(camelCase(category))"
             appearance="flat-button"
             :appearanceOverrides="customCategoryStyles"
-            :disabled="availableRootCategories && !availableRootCategories[val]"
+            :disabled="availableRootCategories &&
+              !availableRootCategories[val] &&
+              !activeKeys.filter(k => k.includes(val)).length"
```

Comment:
Should do this as a computed prop - and there's probably a better way to do the existence check.